kotest-runner/kotest-runner-junit5/src/jvmMain/kotlin/io/kotest/runner/junit/platform/KotestJunitPlatformTestEngine.kt

```diff
@@ -64,17 +69,11 @@ class KotestJunitPlatformTestEngine : TestEngine {
                   request.engineExecutionListener
                ),
                root,
+               engineDescriptor.formatter
             )
          )
       )
 
-      request.configurationParameters.get("kotest.extensions").orElseGet { "" }
-         .split(',')
-         .map { it.trim() }
-         .filter { it.isNotBlank() }
-         .map { Class.forName(it).newInstance() as Extension }
-         .forEach { configuration.registry.add(it) }
-
```

Comment:
💬 Moving this to the discovery phase was necessary since there's an extension that allows customizing display names which are now determined during discovery for spec classes.